src/reset.css

```diff
@@ -0,0 +1,15 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+body h1 p a {
```

Comment:
links dont have default margins, and this selector would not work anyways, use `,` to combine them